### PR TITLE
Css dsl relation

### PR DESCRIPTION
**Pull request by NekoiNemo**
Added ability to use all relations inside CssSelectionBlock's block

### CODE REVIEW

File: src/main/java/tornadofx/CSS.kt [hunk 1]
```diff
@@ -953,13 +952,69 @@ class CssSelectionBlock(op: CssSelectionBlock.() -> Unit) : PropertyHolder(), Se
     @Deprecated("Use and() instead as it's clearer", ReplaceWith("and(selector, *selectors, op = op)"))
     fun add(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit) = and(selector, *selectors, op = op)
 
-    fun and(selector: String, op: CssSelectionBlock.() -> Unit) = and(selector.toSelector(), op = op)
-    fun and(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit): CssSelection {
+    private fun addRelation(
+        relation: CssSubRule.Relation,
+        selector: Selectable, selectors: Array<out Selectable>,
```
Comment (ruckustboom) [inline src/main/java/tornadofx/CSS.kt]: This should be `selector: Selectable, vararg selectors: Selectable` and you would call this function with `addRelation(<relation>, selector, *selectors, op = op)`

File: src/main/java/tornadofx/CSS.kt [hunk 2]
```diff
@@ -953,13 +952,69 @@ class CssSelectionBlock(op: CssSelectionBlock.() -> Unit) : PropertyHolder(), Se
...
+
+    /**
+     * [CssSubRule.Relation.DESCENDANT]
+     */
+    fun contains(selector: String, op: CssSelectionBlock.() -> Unit) = contains(selector.toSelector(), op = op)
+
+    /**
+     * [CssSubRule.Relation.DESCENDANT]
+     */
+    fun contains(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit): CssSelection =
```
Comment (ruckustboom) [inline src/main/java/tornadofx/CSS.kt]: @edvin Do you think we should add the `contains` function? It is more complete, but it is also the default functionality if you don't use a function.

Comment (ruckustboom) [reply src/main/java/tornadofx/CSS.kt]: Never mind, it's good for constancy.

File: src/main/java/tornadofx/CSS.kt (hunk 1 above)
Comment (NekoiNemo) [reply src/main/java/tornadofx/CSS.kt]: Is there any reason to use additional spread operator?

Comment (ruckustboom) [reply src/main/java/tornadofx/CSS.kt]: Yes. For one thing, we've been trying to push using idiomatic code throughout the codebase. For another, it makes the intended use clear and ensures consistency,

**Comment by edvin** [comment]
Perfect, thanks a lot!

**Comment by ruckustboom** [comment]
Looks great! I hope we keep seeing more contributions from you :)

**Pull request merged**